### DISCUSSION TITLE
Javascript repositories

### DISCUSSION THREAD

**Discussion by TomChrister** [community/community#111097] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hey!
I got a repository that should have more javascript in it then html and css, but under languages it says i got like 80% html, 10% css and 10% javascript. What can i do so it shows it's actually more javascript? I would like that it says it's a javascript repository since it's a javascrupt project. Thanks!

Here's a link to my repository:

https://github.com/TomChrister/javascript-ca

**Comment by gsnoopy** [comment, marked as answer]
Github uses Linguist library to detect repository languages, you can create a .gitattributes file in your repository with the below content:

*.html linguist-detectable=false
*.js linguist-detectable=true

then linguist ignores html and shows only javascript, you can add another programming languages.

**Reply by TomChrister** [reply]
Awesome, thanks a lot!
